app/src/ui/changes/files-changed-badge.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+
+interface IFilesChangedBadgeProps {
+  readonly numFilesChanged: number
```

Comment:
I'm not a fan of the shortened `num` here - could we change this to be `filesChangedCount` here to be a bit more descriptive?